components/primitives/ProgressBar/progress-bar.tsx

```diff
@@ -0,0 +1,27 @@
+import clsx from "clsx";
+import { FC, HTMLProps } from "react";
+
+interface ProgressBarProps {
+  value: string; // Provide the progress value as a number (0-100)
+  props?: HTMLProps<HTMLProgressElement>;
+  className?: string;
+}
+
+const ProgressBar: FC<ProgressBarProps> = ({ value, className, ...props }) => {
+  return (
+    <progress
+      className={clsx(
+        className,
+        "appearance-none bg-abeg-neutral-80 h-2 rounded-[4px] w-full border-none mx-9",
+      )}
+      id="target"
+      value={value}
```

Comment:
...convert from number to string at this point yea, like so..

```js
value={string(value)}
```

Reply:
Great catch, although there's now a native HTML way of avoiding to do this, but we can get into those technicalities later🥂

Reply:
Great, I'd like to learn about this later for sure. 😉🤘🏾